components/BottomMenu.js

```diff
@@ -40,6 +43,26 @@ function BottomMenu () {
         );
     }
 
+    if (getInside) {
+        return(
+            <View style={styles.insideBuildingContainer}>
+                <Icon name="ios-arrow-up" style={styles.arrowUp} onPress={() => { setIconSelected(true); }} />
+                <Text style={styles.mainLabel}>{selectedBuilding}</Text>
+                <Text style={styles.shortLabel}>More info</Text>
+                <View style={styles.btnleave}>
+                    <Button style={styles.btnleave} color={"#3ACCE1"} uppercase={false} mode="contained" onPress={() => {
```

Comment:
why aren't you setting the colour inside the styling section?

Reply:
In this case, the Button tag from react-native-paper has color as one of its props. See more [here](https://callstack.github.io/react-native-paper/button.html)